cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -4,15 +4,19 @@ kind: Kustomization
 
 resources:
   - ../common
+  - ../../../../base/cert-manager.io/issuers/ingress-letsencrypt-production
```

Comment:
remove this line since you moved the contents to the other `cert-manager` folder